connectivity/lorawan/tests/UNITTESTS/loramac/unittest.cmake

```
@@ -20,18 +20,18 @@ set(TEST_SUITE_NAME "lorawan_LoRaMac")
 
 # Source files
 set(unittest-sources
-  ../features/lorawan/lorastack/mac/LoRaMac.cpp
+  ../connectivity/lorawan/lorastack/mac/LoRaMac.cpp
 )
 
 # Add test specific include paths
 set(unittest-includes ${unittest-includes}
   target_h
-  ../features/lorawan/lorastack/mac
+  ../connectivity/lorawan/lorastack/mac
 )
 
 # Test & stub files
 set(unittest-test-sources
-  features/lorawan/loramac/Test_LoRaMac.cpp
+  loramac/Test_LoRaMac.cpp
```

Test in the same directory as `unittest.cmake`:
```suggestion
  ${CMAKE_CURRENT_LIST_DIR}/Test_LoRaMac.cpp
```